tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -759,9 +777,21 @@ private async Task<ObjectPackFilesResult> GetObjectsPackFileFromWorkingDirectory
             return new ObjectPackFilesResult(Array.Empty<string>(), temporaryFolder);
         }
 
-        Log.Debug("ITR: Packing objects...");
+        // Sanitize object list (on some cases we get a "fatal: expected object ID, got garbage" error because the object list has invalid escape chars)
+        var objectsOutput = getObjectsCommand!.Output;
+        var lstObjectsSha = ((IList<Match>)ShaRegex.Matches(objectsOutput)).Select(m => m.Value).ToList();
```

Comment:
Tony... using LINQ?! well I never 😛

Reply:
I really didn't want it, it was copilot... 🤣

Reply:
The balance in the universe has been restored.